forge-gui/res/cardsfolder/upcoming/zimone_and_dina.txt

```diff
@@ -0,0 +1,12 @@
+Name:Zimone and Dina
+ManaCost:B G U
+Types:Legendary Creature Human Dryad
+PT:3/4
+T:Mode$ Drawn | ValidCard$ Card.YouCtrl | Number$ 2 | TriggerZones$ Battlefield | Execute$ LoseGain | TriggerDescription$ Whenever you draw your second card each turn, target opponent loses 2 life and you gain 2 life.
+SVar:LoseGain:DB$ LoseLife | ValidTgts$ Opponent | LifeAmount$ 2 | SubAbility$ DBGain
+SVar:DBGain:DB$ GainLife | Defined$ You | LifeAmount$ 2
+A:AB$ Repeat | Cost$ T Sac<1/Creature.Other/another creature> | RepeatSubAbility$ DBDraw | MaxRepeat$ 2 | RepeatCheckSVar$ CheckLands | RepeatSVarCompare$ GE8 | StackDescription$ SpellDescription | SpellDescription$ Draw a card. You may put a land card from your hand onto the battlefield tapped. If you control eight or more lands, repeat this process.
```

Comment:
Use RepeatPresent$ version to avoid extra SVar

Reply:
Fixed